sound/tts/sound.txt

```diff
@@ -886,7 +886,7 @@
     ro = Apoi
     ja = その先
     da = Derefter
-    es = Después
+    es = Después.
     fi = Sitten.
```

Comment:
> @bykoianko ok but do you want me to modify the "fi" language too? 

Yes. 

> This PR was made for the Spanish translation.

I guess that's ok. This point is a mistake and I'd prefer not to create 3 commits for removing only one point if Finish. It's optional. But it would be nice if you remove this point.